azure-pipelines.cache.yml

```diff
@@ -2,9 +2,9 @@
 steps:
   - task: Cache@2
     inputs:
-      key: 'gulp-cache | "$(Agent.OS)"'
+      key: 'gulp-cache-v001 | "$(Agent.OS)"'
       restoreKeys: |
-        gulp-cache | "$(Agent.OS)"
-        gulp-cache
+        gulp-cache-v001 | "$(Agent.OS)"
+        gulp-cache-v001
```

Comment:
Curious why this is needed? Is there now a permanent `gulp-cache` store somewhere?

Reply:
The gulp cache in the OS temp directory isn't being cleared between builds.

Reply:
And this PR's build was still failing due to what we assume is a previous invalid cache.

Reply:
Oops, totally missed there were multiple commits.